app/views/internal/configs/show.html.erb

```diff
@@ -144,23 +144,14 @@
                      } %>
           <div id="contentBodyContainer" class="card-body collapse hide" aria-labelledby="contentBodyContainer">
             <div class="form-group">
-              <%= f.label :community_description, "Community description" %>
+              <%= f.label :community_description, "Community description*" %>
```

Comment:
I'm afraid that if we add the stars for mandatory fields manually they might get out of sync. Maybe we can make a custom label helper that checks against `VerifySetupCompleted::MANDATORY_CONFIGS`, I'll take a stab at that after finishing the review.